utils/cliutils/commandsflags.go

```diff
@@ -2024,6 +2025,9 @@ var commandFlags = map[string][]string{
 		platformUrl, user, password, accessToken, serverId, lcPathMappingTarget, lcPathMappingPattern, Project,
 		downloadMinSplit, downloadSplitCount,
 	},
+	ReleaseBundleImport: {
+		user, password, accessToken, serverId, lcDryRun,
```

Comment:
Looks like we are not actually supporting the dry run capability and this flag isn't used. Lets add the functionality throughout the code.
Also, missing `platformUrl` here